src/app/search/search-results/search-results.component.ts

```diff
@@ -124,9 +126,17 @@ export class SearchResultsComponent implements OnInit, OnDestroy {
     if (this.pageConfig.displayDocumentLabelField) {
       this.displayedColumns.push('label');
     }
+
     for (const field of this.pageConfig.fieldsToDisplay) {
       this.displayedColumns.push(field.key);
+      this.fieldLabelMap[field.key] = field.label;
     }
+
+    this.matSortService.sortButtonLabel = id => {
```

Comment:
Would it be possible to add the type?
ex:
```suggestion
    this.matSortService.sortButtonLabel = (id: string) => {
```